lib/chef/http/ssl_policies.rb

```diff
@@ -129,5 +129,17 @@ def set_verify_mode
       end
     end
 
+    class VerifyPeerSSLPolicy < DefaultSSLPolicy
```

Comment:
Can we get at least a basic comment here on the intent of these classes and where they'll get used. Help future folks out. I realize this entire file is uncommented, but we need to start somewhere